MIT.md

```diff
@@ -1,10 +1,7 @@
-## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] Lastly, this information was supposed to be modified. This will render your readme file professional :+1: